xarray/backends/cfgrib_.py

```diff
@@ -20,7 +21,9 @@
     has_cfgrib = True
 except ModuleNotFoundError:
     has_cfgrib = False
-
+except Exception as e:
```

Comment:
```suggestion
except RuntimeError:
```

you should be as specific as possible.

Reply:
I know, but `cfgrib` may also raise AttributeError is the version of `eccodes` is does not support an expected symbol.

Reply:
You can combine errors `except (RuntimeError, AttributeError):`